backend/sources/gobgp/utils.go

```diff
@@ -0,0 +1,21 @@
+package gobgp;
+
+import (
+	// Standard imports	
+	"crypto/sha1"
+	"io"
+	"fmt"
+
+	// External imports
+	api "github.com/osrg/gobgp/api"
+
+	// Internal imports
+)
+
+func PeerHash(peer *api.Peer) (string) {
+	h := sha1.New()
+	io.WriteString(h, string(peer.State.PeerAs))
+	io.WriteString(h, peer.State.NeighborAddress)
+	sum := h.Sum(nil)
+	return fmt.Sprintf("%x",sum[0:5])
```

Comment:
I'm checking if there might be a better hashing algorithm... I guess this works because sha1 has good diffusion properties, but I wonder if there is not something less prone to collisions (as we are only taking the first 5 byte of the result)

Reply:
Put it on a todo this should work for now